src/pages/NewSignup.vue

```diff
@@ -300,16 +305,16 @@ export default {
         redirectToDestination(
           this.$store.state.sessionData.purpose.params,
           this.userData["user_id"],
-          this.$store.state.sessionData.platform_id,
-          this.$store.state.sessionData.platform,
+          this.platform_id,
+          this.platform,
           this.$store.state.groupData.input_schema.userType
         )
       ) {
         sendSQSMessage(
           "attendance-sign-up",
           this.$store.state.sessionData.purpose["sub-type"],
-          this.$store.state.sessionData.platform,
-          this.$store.state.sessionData.platform_id,
+          this.platform,
+          this.platform_id,
           this.userData["user_id"],
           "",
```

Comment:
what is this for? can u add besides it in comment

Reply:
Done